### ISSUE TITLE
Cannot type accent mark [á, é...]

### ISSUE THREAD

**Issue by coriumalpha** [telegramdesktop/tdesktop#1977]
### Steps to reproduce
1. Try to write on "Search" or "Write a message..." boxes.
2. Press the accent mark key (then, release). Press the letter to be written (release).
### Expected behaviour

After step 2, the letter must be written with the accent mark (á, é, í, ó, ú).
### Actual behaviour

Actually, after step 2, the letter is shown without any accent.
### Configuration

**Operating system:** Ubuntu 15.10 Gnome 3.16

**Version of Telegram Desktop:** 0.9.46 alpha

**Comment by auchri** [comment]
duplicate #1041 ?

**Comment by coriumalpha** [comment]
I don't think, it is happening on more than one PC (tested on Ubuntu 15.10 & 16.04) and just after updating to 0.9.46. I downgraded to the previous version and everything works fine.

**Comment by axd** [comment]
Same issue here.

Xubuntu 16.04 
Telegram 0.9.49

**Comment by vicerodrigues** [comment]
Same here...

Ubuntu 16.04 Unity
Telegram 0.9.51

**Comment by bertob** [comment]
Same here.

Arch Linux with GNOME 3.20
Telegram 0.9.56

**Comment by rhaps0dy** [comment]
Possibly duplicate of #1041. narcelio published a workaround there:

> Workaround: call with `QT_IM_MODULE=xim /opt/telegram/Telegram`

It fixed the issue for me, on Xubuntu 16.04.
